### PR TITLE
Add story for scrollTarget

### PR DESCRIPTION
**Pull request by jyash97**
Requires [#537](https://github.com/appbaseio/reactivesearch/pull/537)

### CODE REVIEW

File: stories/reactivesearch/ReactiveList.stories.js [hunk 1]
```diff
@@ -5,6 +5,10 @@ import { booksReactiveList } from "./resultViews";
 
 export default class ReactiveListDefault extends Component {
 	render() {
+		const scrollStyle = this.props.scrollTargetStory ? ({
```
Comment (metagrover) [inline stories/reactivesearch/ReactiveList.stories.js]: You will need to destructure other props now. `this.props.scrollTargetStory` is getting passed to `ReactiveList` on Line 38.

**Review by metagrover — CHANGES_REQUESTED** [review]

**Review by metagrover — APPROVED** [review]

**Comment by jyash97** [comment]
@metagrover Should I remove `yarn.lock` file from this ?

**Comment by metagrover** [comment]
Does this lock file fix the deployment issue, where we needed to upgrade
the storybook?

**Comment by jyash97** [comment]
Not sure, I just made this changes with playground cloned repo. Should I try deploying this ?

**Pull request merged**